test/tasks/test_python_grpc.py

```diff
@@ -22,33 +22,33 @@
 from artman.tasks import python_grpc_tasks
 
 
-def test_get_subdir_path():
+def test_get_gapic_subdir_path():
     task = python_grpc_tasks.PythonMoveProtosTask()
     with mock.patch.object(os, 'walk') as walk:
         walk.return_value = (
-            ('foo', ['bar'], []),
-            ('foo/bar', ['proto'], []),
-            ('foo/bar/proto', [], ['file_pb2.py']),
+            ('project', ['v1'], []),
```

Comment:
Can we add a test that covers new functionality (the `docs` and `tests` directory that contains `gapic` subdirectory and it not returned by `get_gapic_subdir_path`?

Reply:
It doesn't matter if `docs` or `tests` include a gapic folder, `os.walk` will never be run on them. I've added a mock of `os.listdir` showing this.